src/Pos/Txp/Core/Types.hs

```diff
@@ -152,10 +152,16 @@ instance Buildable TxOut where
     build TxOut {..} =
         bprint ("TxOut "%coinF%" -> "%build) txOutValue txOutAddress
 
-type TxOutAux = (TxOut, [(StakeholderId, Coin)])
+-- | Transaction output and auxilary data corresponding to it.
+-- [CSL-366] Add more data.
```

Comment:
TODO: fix description of `TxOutAux` in docs in `binary-protocols` (and also after CSL-366).
Also, how about changing `TxAux` from triple to data type with fields?